OpenSim/Common/Test/testSerialization.cpp

```diff
@@ -81,8 +81,98 @@ static void dumpObj(const Object& obj, int nSpaces) {
     }
 }
 
+static void testPropertyOutputHelper(const double val, const std::string ans)
```

Comment:
No reason except that I forgot. The latest commit should add pass by reference for types that are non-basic (I couldn't remember what our style is for passing by reference on basic types?)

Reply:
> I couldn't remember what our style is for passing by reference on basic types?

I don't think we have an agreed-upon style for this.

Reply:
I generally like `const T&` for const input arguments. For simple types it rarely matters but gets us in the habit of using references so when it does matter we don't have to fix it.

Reply:
IIUC you don't gain much, if anything, by doing `const double& val` since the address is as long as the data itself—but I like using `const` wherever possible and I think it should be `const std::string& ans` (call by reference for string) on L84.